src/frontend/src/components/FilterPanel/components/DateFilter/index.tsx

```diff
@@ -64,6 +68,23 @@ const DateFilter: FC<Props> = ({
     setAnchorEl(undefined);
   };
 
+  const validationSchema = useMemo(
```

Comment:
nice, we should be using useMemo in a lot of other places too.. lol

Reply:
Credit where it's due, the useMemo is all Timmy. He pushed up a branch with an alternative, better fix to the `isValid` visual flicker problems I was having, and this was one of his other improvements.

Reply:
@tihuan 💯 😸 🚀